debian/authd.service.in

```diff
@@ -26,6 +28,63 @@ RestrictRealtime=yes
 RestrictSUIDSGID=yes
 SystemCallArchitectures=native
 
+# This always corresponds to /var/cache/authd
+CacheDirectory=authd
+CacheDirectoryMode=0700
+
+# This always corresponds to /var/lib/authd
+StateDirectory=authd
```

Comment:
Oh, `StateDirectory`! This will definitively be used as variable names in future projects :)

Reply:
Yes, we could also rely on `$STATE_DIRECTORY` env variable in code to be cleaner, but there's no need for now.